Microsoft.Toolkit.Uwp.UI.Behaviors/Keyboard/KeyDownTriggerBehavior.cs

```diff
@@ -0,0 +1,66 @@
+using System.Windows.Input;
+using Microsoft.Xaml.Interactivity;
+using Windows.System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Behaviors
+{
+    /// <summary>
+    /// This behavior listens to a key down event on the associated <see cref="UIElement"/> when it is loaded and executes an action.
+    /// </summary>
+    [TypeConstraint(typeof(FrameworkElement))]
+    public class KeyDownTriggerBehavior : Trigger<UIElement>
```

Comment:
If we have a constraint that it should be a FrameworkElement, can we not just make it a `Trigger<FrameworkElement>` instead? Is there a reason it doesn't work on a general UIElement?

Reply:
Fixed to -> Trigger<FrameworkElement>